client/js/serverstate.js

```diff
@@ -58,6 +58,8 @@ export function addWidget(widget, instance) {
     w = new Spinner(id);
   } else if(widget.type == 'seat') {
     w = new Seat(id);
+  } else if(widget.type == 'scoreboard') {
+    w = new Scoreboard(id);
```

Comment:
abCdE :)

Reply:
You and @ArnoldSmith86 can fight this one out. I had it that way to begin with.

Reply:
If I recall it right you had it fully flexible right?
Not respecting the wight and height but making everything visible

This is exactly the opposite. The only thing respected is width and height.

I'm just suggesting to add the option to make the cells fit the size. I think the css unit fr should pretty much do that by itself.

Reply:
But how is this even an option? 😆

Reply:
Took me a while but Raphael is saying the widget types are sorted alphabetically and "scoreboard" should be before "seat".

Reply:
> Took me a while but Raphael is saying the widget types are sorted alphabetically and "scoreboard" should be before "seat".

What? I'm sorry! I thought it was clear 😂

I'm really sorry for the confusion. Yeah it was. Just a funny way of saying that sCoreboard should come before sEats.

Also o checked in others places and this was the only that I saw this.

Reply:
That entire section is *not* in alphabetical order *at all*. Just saying. Happy to alphabetize the whole thing.
```
 } else if(widget.type == 'pile') {
    w = new Pile(id);
  } else if(widget.type == 'canvas') {
    w = new Canvas(id);
  } else if(widget.type == 'deck') {
    w = new Deck(id);
  } else if(widget.type == 'holder') {
    w = new Holder(id);
  } else if(widget.type == 'spinner') {
    w = new Spinner(id);
  } else if(widget.type == 'seat') {
    w = new Seat(id);
  } else if(widget.type == 'scoreboard') {
    w = new Scoreboard(id);
  } else if(widget.type == 'timer') {
    w = new Timer(id);
  } else if(widget.type == 'label') {
    w = new Label(id);
  } else if(widget.type == 'button') {
    w = new Button(id);
  } else {
    w = new BasicWidget(id);
  }
```

Reply:
Just ignore it then. I didn't look at more than the code GitHub shows for this comment.

Reply:
Oh yeah... that is terrible

Yeah sorry I only looked in the changed area

Reply:
> Just ignore it then. I didn't look at more than the code GitHub shows for this comment.

@ArnoldSmith86 how about no...
https://github.com/ArnoldSmith86/virtualtabletop/pull/1444

Reply:
Merged and resolved this problem